### ISSUE TITLE
Cut and Paste Leaves Almost-Empty File in Original Location

### ISSUE THREAD

**Issue by cr45hmurphy** [OllisGit/OctoPrint-DisplayLayerProgress#208]
Salandora/OctoPrint-FileManager#37

**Comment by HedgesHandmade** [comment]
I'm experiencing the same issue as it relates to the DeleteAfterPrinting plug-in.  I have it set to move the file to the _archive folder after successfully printing, which it does successfully, but it also leaves the same 130 byte file with the same filename in the main file list.  The text inside the file reads:

```
; BEGIN DISPLAYLAYERPROGRESS SETTINGS
; DisplayLayerProgress_layerIndicatorProcessed = true
; END   DISPLAYLAYERPROGRESS SETTINGS

```

**Comment by OllisGit** [comment]
similar to #207

**Comment by OllisGit** [comment]
Hi @cr45hmurphy! 
Fixed in lates release V1.25.4

Please test and give feedback.

Thx, in advance!
Olli

**Comment by cr45hmurphy** [comment]
> 
> 
> Hi @cr45hmurphy, @HedgesHandmade !
> Fixed in lates release V1.25.4
> 
> Please test and give feedback.
> 
> Thx, in advance!
> Olli

Updating now. Will provide feedback shortly.

**Comment by cr45hmurphy** [comment]
It appears to be fixed!

**Comment by HedgesHandmade** [comment]
> Hi @cr45hmurphy, @HedgesHandmade !
> Fixed in lates release V1.25.4
> 
> Please test and give feedback.
> 
> Thx, in advance!
> Olli


Working perfectly now!
